azure-pipelines.yml

```diff
@@ -30,21 +30,26 @@ resources:
 
 variables:
   - template: '/miscellaneous/build_templates/variables.yml@templates'
-  - name: POLARIS_PROJECT_NAME
-    value: SDS_Python
-  - name: BLACK_DUCK_PROJECT_NAME
+  - name: analysisProject
     value: SDS_Python
 
 parameters:
+  - name: poolADH
+    default: 00-OSIManaged-Containers
   - name: containerDemandsADH
-    default: Agent.OS -equals Linux
+    type: object
+    default: ['Agent.OS -equals Linux]
+  - name: poolEDS
+    default: 00-OSIManaged-Build
   - name: containerDemandsEDS
-    default: PYTHON
+    type: object
+    default: ['PYTHON']
```

Comment:
There seems to be an issue with the changes related to the ['Python'] here.

From the pipeline 
![image](https://user-images.githubusercontent.com/92155840/184100298-e24ade57-c4c6-4f25-ad45-a8ee7c1f529e.png)